app.js

```diff
@@ -0,0 +1,17 @@
+const { getTopics } = require("./controllers/topics.controller.js");
+const express = require("express");
+const res = require("express/lib/response");
+const app = express();
+
+app.use(express.json());
```

Comment:
If David were reviewing your code, he'd call you out on using this before you need it. Just make sure you're 100% clear on what this line is doing before you move on, and your secret is safe with me.